### ISSUE TITLE
Translate the complete ERROR message to the VCH creation wizard upon failure for easy interpretation

### ISSUE THREAD

**Issue by lgayatri** [vmware/vic-ui#291]
Build:  https://storage.cloud.google.com/vic-product-ova-releases/vic-v1.3.0-f8cc7317.ova

./vic-machine-linux --version
vic-machine-linux version v1.3.0-15552-473375a

./vic-ui-linux --version
vic-ui-linux version 15552-473375a0a80f4f91ac445ecb616056fc5cfcbcc7

![image](https://user-images.githubusercontent.com/25006749/34598676-2045eb82-f214-11e7-805b-2a3dcadb207b.png)

We need to add more detail to the error message that is printed here for the user to be able to interpret the issue.

Command generated:
```
./vic-machine-linux create --name virtual-container-host --compute-resource /Datacenter/host/Cluster --image-store CPBU_2TB_01 --base-image-size 8GB --bridge-network vxw-dvs-3444-virtualwire-1-sid-5000-VCH-LS-1 --bridge-network-range 172.16.0.0/12 --public-network DPortGroup --public-network-ip 10.197.37.192/23 --public-network-gateway 10.197.37.253 --dns-server 10.142.7.21 --tls-cname virtual-container-host --organization virtual-container-host --certificate-key-size 2048 --no-tlsverify --user Administrator@VSPHERE.LOCAL --thumbprint C1:6D:3E:41:A5:89:5E:89:FB:DB:82:2F:32:82:8E:54:53:8B:65:34 --target wdc-stvc3944-146-dhcp15.eng.vmware.com/Datacenter --ops-user lgayatri@vsphere.local --ops-grant-perms
```

@mdubya66 @AngieCris @jak-atx -> Enhancement request.

**Comment by lgayatri** [comment]
Looks like the browser session expired during VCH creation

**Comment by AngieCris** [comment]
Issue for H5 session expire: https://github.com/vmware/vic-ui/issues/228

**Comment by jak-atx** [comment]
Closing as dupe, to be addressed in #228
